Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100% |[Finetuning] Epoch: 1 Avg Loss: 0.3136[Validation] Avg Loss: 0.3288, Avg Dice Score: 0.8959100%[Finetuning] Epoch: 2 Avg Loss: 0.3035[Validation] Avg Loss: 0.3269, Avg Dice Score: 0.8976100%[Finetuning] Epoch: 3 Avg Loss: 0.2998[Validation] Avg Loss: 0.3357, Avg Dice Score: 0.8823100%[Finetuning] Epoch: 4 Avg Loss: 0.2965[Validation] Avg Loss: 0.3354, Avg Dice Score: 0.8830100%[Finetuning] Epoch: 5 Avg Loss: 0.2933[Validation] Avg Loss: 0.3399, Avg Dice Score: 0.8772100%[Finetuning] Epoch: 6 Avg Loss: 0.2952[Validation] Avg Loss: 0.3341, Avg Dice Score: 0.8929100%[Finetuning] Epoch: 7 Avg Loss: 0.2946[Validation] Avg Loss: 0.3354, Avg Dice Score: 0.8836100%[Finetuning] Epoch: 8 Avg Loss: 0.2927[Validation] Avg Loss: 0.3277, Avg Dice Score: 0.8980100%[Finetuning] Epoch: 9 Avg Loss: 0.2886[Validation] Avg Loss: 0.3294, Avg Dice Score: 0.8988 100%||[Finetuning] Epoch: 10 Avg Loss: 0.2860[Validation] Avg Loss: 0.3277, Avg Dice Score: 0.9012 /home/mikhelson/Med143/143 [00:14<00:00, 10.05it/s]143/143 [00:14<00:00, 10.13it/s]143/143 [00:14<00:00, 10.16it/s]143/143 [00:13<00:00, 10.35it/s]143/143 [00:13<00:00, 10.34it/s]143/143 [00:13<00:00, 10.26it/s]143/143 [00:14<00:00, 10.15it/s]143/143 [00:14<00:00, 10.19it/s]143/143 [00:14<00:00, 10.10it/s]143/143 [00:13 00:00, 10.23it/s]ImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to `self.log()` but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'Train Test Results - Average Loss: 0.6160, Average Dice Score: 0.7360er'